metastore/scripts/upgrade/derby/058-HIVE-23516.derby.sql

```diff
@@ -4,7 +4,7 @@ CREATE TABLE "APP"."REPLICATION_METRICS" (
   "RM_POLICY" varchar(256) NOT NULL,
   "RM_DUMP_EXECUTION_ID" bigint NOT NULL,
   "RM_METADATA" varchar(4000),
-  "RM_PROGRESS" varchar(4000),
+  "RM_PROGRESS" varchar(24000),
```

Comment:
these files are older ones. You can skip updating them. Only the scripts inside standalone-metastore should be updated.